Common/triggerSel.py

```diff
@@ -0,0 +1,105 @@
+import ROOT
+import Common.BaselineSelection as Baseline
+syst_names = {"_Central":Baseline.ana_reco_object_collections, "_TauESUp":["Tau", "MET"], "_TauESDown":["Tau", "MET"]}
+
+
+def ApplyTriggers(df, yaml_dict= None, isData = False):
```

Comment:
Implement with `class` and load config in the constructor

Reply:
ok